baselines/hfedxgboost/hfedxgboost/conf/clients/cod_rna_5_clients.yaml

```diff
@@ -0,0 +1,9 @@
+n_estimators_client: 50
```

Comment:
Shouldn't be the number of estimators (`M`) in the paper be equal to 500/num_clients? (this applies to other configs also). I think this hyperparameter should be kept as in the paper.

Reply:
gone through that during our last meeting